src/types/index.d.ts

```diff
@@ -390,7 +390,10 @@ export interface DataPlane extends MeshEntity {
  */
 export interface DataPlaneOverview extends MeshEntity {
   type: 'DataplaneOverview'
-  labels?: Record<string, string>
+  labels?: {
+    'kuma.io/display-name'?: string
+    'k8s.kuma.io/namespace'?: string
```

Comment:
**Suggestion**: This change would prevent accessing other properties. It should be:

```suggestion
    'k8s.kuma.io/namespace'?: string
    [key: string]: string | undefined
```

Reply:
There's something to be said for not letting ourselves statically accessing properties that aren't specified here but 🤷